R/splits_functions.R

```diff
@@ -36,29 +36,35 @@
 #'
 #' @export
 #'
-calcRate <- function(M, Aj, data_SS = NULL, days = NULL, searches = NULL){
+calcRate <- function(M, Aj, days = NULL, searches_carcass = NULL, data_SS = NULL){
  if (!is.null(data_SS) && ("SS" %in% class(data_SS))){
     days <- data_SS$days
-    searches <- data_SS$searches
+    unit <- rownames(Aj)
+    searches_carcass <- array(0, dim = c(x, length(days)))
+    for (xi in 1:x) searches_carcass[xi, ] <- data_SS$searches_unit[unit[xi], ]
+  } else if (!is.null(data_SS)){
+    stop(
+      "In arg list, data_SS must be an SS object. Alternatively, user may ",
+      "provide 'days' and 'carcass_searches'."
+    )
   }
   if (is.vector(M)){
     M <- matrix(M, nrow = 1)
     Aj <- matrix(Aj, nrow = 1)
     searches <- matrix(searches, nrow = 1)
```

Comment:
should this be 
`searches_carcass <- matrix(searches_carcass, nrow = 1)`?

Reply:
yup. good catch.